acme.sh

```diff
@@ -491,13 +491,13 @@ _post() {
     CURL="$CURL --dump-header $HTTP_HEADER "
     if [ "$needbase64" ] ; then
       response="$($CURL -A "User-Agent: $USER_AGENT" -X POST --data "$body" $url | _base64)"
-      if [[ $? -gt 0 ]] ;then
-         _err "Curl failed with exit code $?, trying running script in DEBUG=2 mode"
+      if [ $? -gt 0 ] ;then
```

Comment:
In this `post()` function, please add a `ret` variable to use, just as we did in the `_get()` function.
Thanks.